Packs/EWS/Integrations/EWSv2/EWSv2.py

```diff
@@ -1103,87 +1104,95 @@ def parse_incident_from_item(item, is_fetch):
+                                    if attachment.item.headers:
+                                        attached_email_headers = []
+                                        for h, v in attached_email.items():
+                                            if not isinstance(v, str):
+                                                try:
+                                                    v = str(v)
+                                                except:     # noqa: E722
+                                                    demisto.debug('cannot parse the header "{}"'.format(h))
+                                                    continue
+
+                                            v = ' '.join(map(str.strip, v.split('\r\n')))
+                                            attached_email_headers.append((h, v))
+
+                                        for header in attachment.item.headers:
+                                            if (header.name, header.value) not in attached_email_headers \
+                                                    and header.name != 'Content-Type':
+                                                attached_email.add_header(header.name, header.value)
+
+                                    file_result = fileResult(get_attachment_name(attachment.name) + ".eml",
+                                                             attached_email.as_string())
+
+                                if file_result:
+                                    # check for error
+                                    if file_result['Type'] == entryTypes['error']:
+                                        demisto.error(file_result['Contents'])
+                                        raise Exception(file_result['Contents'])
+
+                                    # save attachment to incident
+                                    incident['attachment'].append({
+                                        'path': file_result['FileID'],
+                                        'name': get_attachment_name(attachment.name) + ".eml"
+                                    })
+
+                                else:
+                                    incident['attachment'].append({
+                                        'name': get_attachment_name(attachment.name) + ".eml"
+                                    })
+
+                            labels.append({'type': label_attachment_type, 'value': get_attachment_name(attachment.name)})
+                            labels.append({'type': label_attachment_id_type, 'value': attachment.attachment_id.id})
+                        except ErrorCannotOpenFileAttachment:
+                            demisto.error("Could not open file attachment file for message_id: {}".format(item.message_id))
+        except ErrorCannotOpenFileAttachment:
+            demisto.error("Could not open file attachment file for message_id: {}".format(item.message_id))
```

Comment:
can we be minimize the scope of the try-except?
relevant for both integrations
```suggestion
            demisto.error("Could not open attachment file for message_id: {}".format(item.message_id))
```